portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1144,6 +1147,7 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, geneset_ids,
 		if (data_by_id[geneset_id][case_id].profile_data === null) {
 		    // set the initial value for this sample or patient
 		    data_by_id[geneset_id][case_id].profile_data = sample_score;
+		    data_by_id[geneset_id][case_id].na = false;
```

Comment:
@adamabeshouse Following the original example in https://github.com/cBioPortal/cbioportal/pull/1847, this loop iterates over the scores received from the API to add them to the appropriate records, and leaves the others untouched at the default of `{profile_data: null, na: true}`. I believe that deleting properties may be less efficient in some browsers than simply assigning. Would you prefer if I assigned `undefined` instead of `false`?